core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycleSpec.scala

```diff
@@ -154,14 +154,11 @@ class BpmBillingProjectLifecycleSpec extends AnyFlatSpec {
     when(repo.setBillingProfileId(createRequest.projectName, profileModel.getId)).thenReturn(Future.successful(1))
 
     val wsmResouceRecordDao = mock[WorkspaceManagerResourceMonitorRecordDao]
-    when(
-      wsmResouceRecordDao.create(
-        landingZoneJobId,
-        JobType.AzureLandingZoneResult,
-        createRequest.projectName.value,
-        testContext.userInfo.userEmail
-      )
-    ).thenReturn(Future.successful())
+
+    doReturn(Future.successful())
```

Comment:
This means we lose checking that the correct information was stored. I would prefer checking the expected arguments.

Reply:
I'll see what I can do. Having "smart constructors" has cost us the ease of doing this though I think it's a good compromise to make sure we're constructing objects with the right state.

Reply:
I've added back some argument verification